.github/workflows/test-sys.yaml

```diff
@@ -206,7 +206,7 @@ jobs:
         if: matrix.run_test && matrix.os != 'windows-2019'
         shell: bash
         run: |
-          make build-wasmer && make build-capi && make package-capi && make package && export WASMER_DIR=`pwd`/package && make test-integration-cli
+          curl https://get.wasmer.io -sSfL | sh && make build-wasmer && make build-capi && make package-capi && make package && export WASMER_DIR=`pwd`/package && make test-integration-cli
```

Comment:
I'd like not to depend on wapm on here. Mainly, because we are moving away from it, and it introduces technical debt

Reply:
To be specific: I'd not to like to depend on wapm in any case. We should see what command is wapm doing that make wasmer to fail and start from that case.

Reply:
Done, but we should have a check / test suite that runs before every release (not on every PR / commit) that tests the whole ecosystem: wasmer-js, wapm-cli, wasmer-python, etc. so that this doesn't happen again.